src/middlewared/middlewared/plugins/chart_releases_linux/portals.py

```diff
@@ -41,22 +41,22 @@ def retrieve_portals_for_chart_release(self, release_data, node_ip=None):
             return self.PORTAL_CACHE[release_data['name']]
 
     @private
-    def retrieve_portals_for_chart_release_impl(self, release_data, node_ip=None):
+    def retrieve_portals_for_chart_release_impl(self, release_data, node_ip):
         questions_yaml_path = os.path.join(
             release_data['path'], 'charts', release_data['chart_metadata']['version'], 'questions.yaml'
         )
         if not os.path.exists(questions_yaml_path):
             return {}
 
+        if release_data['chart_metadata']['name'] == 'ix-chart':
+            return self.get_ix_chart_portal(release_data, node_ip)
```

Comment:
There is no reason to limit this to ix-chart here

Reply:
I don't follow, can you clarify ?

Reply:
There is no reason this should be made "iX-Official" (hence ix-chart) only.

Reply:
Why ? Other apps can already define as many portals as they want ?

Reply:
No they cannot, not dynamically.

We had to implement weird hacks using configmaps to implement basically this feature and even so we cannot dynamically enable or disable them from values.yaml.

Reply:
Are you talking about users configuring port or protocol or is it something else because i still don't follow you ? And if it's port/protocol, we still go with what we already have even in the official apps because i'm not seeing any motivation to change that workflow unless we have a good use case to do so ?

Reply:
Making this iX-Only, us simply not needed in any way, shape or form. There is no technical reason to do so. This seems highly suspicious to be frank, as it simply does not make any sense to have a simple values.yaml parser locked behind an "ix-chart" if statement.

It's on you to explain why this so harshly needs to be ix-locked, as you are proposing to ix-lock this feature.

---

As I already explained: we really need this to be able to simplify our portal generator. I'm not going to write a long tedious writedown of our codebase, as it's opensource and we both have beter things to do.

Questions.yaml is not accessable using helm templating, values.yaml is.
We write our charts using a lot more helm templating functionality than every would be possible using the questions.yaml system/file.

Hence being able to set things in values.yaml allows us a LOT more flexibility, as it gives us access to those variables using the native helm templating engine directly.